Networking/WebSockets/WebSocketInterface.hh

```diff
@@ -83,8 +83,9 @@ namespace litecore { namespace websocket {
         kNetErrHostUnreachable,
         kNetErrAddressNotAvailableAIL,
         kNetErrBrokenPipe,
+        kC4NetErrUnknownInterface,
         // Add new codes here. You MUST add messages to kLiteCoreMessages!
-        // You MUST add corresponding kC4NetErr codes to the enum in C4Base.h!
+        // You MUST add corresponding kC4NetErr codes to the enum in c4Error.h!
```

Comment:
Not sure about adding message to kLiteCoreMessages. It seems like that should be done as well.

Reply:
added inside kNetworkMessages as is a NetErr